WebApplication1/WebApplication1/mainTests.js

```diff
@@ -0,0 +1,27 @@
+﻿/// <reference path="scripts/jasmine.js" />
+/// <reference path="main.js" />
```

Comment:
The test's are mostly irrelevant, they don't test the solution.

I expect to have a test that receives a predefined output of and checks that the output is ok, Something along this lines.

```js
// prepare
var location1 = new Location('Location1', 47.5809, -122.3160);
var location2 = new Location('Location2', 37.5209, -122.3340);

// execute
var results = solution.check(47.6, -122.4, [location2, location1]);

// assert
expect(results[0].name).toBe('Location1');
expect(results[1].distance).toBe(0.0645);
```